### ISSUE TITLE
cwltool-output.json contains logging and json

### ISSUE THREAD

**Issue by johnbradley** [Duke-GCB/lando#89]
Creating a report failed due to inability to parse cwltoil stdout (which should be in json format).
```
2017-10-16 07:00:55,203 - toil.batchSystems.singleMachine - DEBUG - Exiting worker thread normally.
2017-10-16 07:00:55,287 - toil.common - DEBUG - Obtained node ID f2be0a26b6a94d2cb03edd5151ab84f8 from file /var/lib/dbus/machine-id
2017-10-16 07:00:55,316 - toil.common - DEBUG - ... finished shutting down the batch system in 0.51638007164 seconds.
{
    "bams_markduplicates_dir": {
        "class": "Directory",
        "basename": "bams-markduplicates",
        "location": "file:///work/data_for_job_49/working/results/bams-markduplicates",
        "listing": [

```

This caused an error:
```
  File "/usr/local/lib/python2.7/dist-packages/lando/worker/cwlworkflow.py", line 289, in _create_report
    workflow_info.update_with_job_output(job_output_path=os.path.join(logs_directory, JOB_STDOUT_FILENAME))
  File "/usr/local/lib/python2.7/dist-packages/lando/worker/cwlreport.py", line 165, in update_with_job_output
    doc = parse_yaml_or_json(job_output_path)
  File "/usr/local/lib/python2.7/dist-packages/lando/worker/cwlreport.py", line 275, in parse_yaml_or_json
    doc = yaml.load(infile)
...
ScannerError: mapping values are not allowed here
  in "data_for_job_49/working/results/docs/logs/cwltool-output.json", line 2, column 68
```

**Comment by johnbradley** [comment]
I can reproduce this with a toy workflow using no additional options in toil 3.11.0.
If I switch to the previous version of toil (3.10.1) this issue goes away.

**Comment by dleehr** [comment]
I see. I wonder if disabling all the debug logging would help this.

**Comment by dleehr** [comment]
I updated toil from 3.10.1 to 3.11.0 when redeploying on Friday.

**Comment by johnbradley** [comment]
Adding `--logOff` to the command line using the latest version of toil fixes this issue.

**Comment by johnbradley** [comment]
I really wish there was an option to save the job json output directly to a file.

**Comment by dleehr** [comment]
I assume `--logOff` conflicts with `--logDebug`. Would the `--logFile` still be written?

**Comment by johnbradley** [comment]
```
  --logOff              Same as --logCritical
  --logCritical         Turn on logging at level CRITICAL and above. (default
                        is INFO)
```

**Comment by dleehr** [comment]
We'll revert to 3.10.1 since there's no obvious fix in 3.11
